Add files via upload

dncsilla opened the pull request:
DB Recruitment Agency homework

dncsilla committed:
Add files via upload

DB Recruitment Agency homework

glybinahanna reviewed (changes requested):
Great job! Very good schema and clear description. Here are few comments:
- please make sure that tables in columns on the schema and in the description are the same. For example, Resume column of Candidates table is missing on the schema, but mentioned in description.
- if a table has a composite PK, please mention this in the description (on the current schema it refers to bridge tables).
- JobSkills and CandidateSkills tables violate 3NF: Skills column doesn't directly depend on CandidateSkillID, if different candidates have same skill, this skill will be repeated several times under different CandidateSkillID. In both cases you need to establish many-to-many relation between Candidate or Job and Skills. This also can be implemented similar to what you've done for Statuses (one table with all statuses referring to different tables).

**Not Approved. Please, fix based on my feedback.**

dncsilla commented:
Hi Hanna, 

Thank you very much for your review and comments ! :) 
1, I went through all the columns - schema differences, I corrected it in case of the Candidates table and also in case of the Jobs and Interviews table.
2, I have expanded the description in the word document in the 'Comments of table relationships' section according to your comment.
3, Created Skills table and its relations. CandidateSkills and JobSkills table was updated to join tables. 
Word document and the schema is also updated accordingly. :)
[DB_Csilla_Dan_HW_RecruitmentAgency_descriptions.docx](https://github.com/user-attachments/files/17477385/DB_Csilla_Dan_HW_RecruitmentAgency_descriptions.docx)
![DB_Csilla_Dan_HW_RecruitmentAgency_diagram](https://github.com/user-attachments/assets/c7953c7d-abcb-4cb0-bfe2-8ad6f25cd559)

glybinahanna reviewed (approved):
Thank you for quick fixes :)
Great job! Approved, please merge the branch.
